src/Config.php

```diff
@@ -290,4 +294,23 @@ public function setExportOrdernumberVariantBarcodes(bool $exportOrdernumberVaria
     {
         $this->exportOrdernumberVariantBarcodes = $exportOrdernumberVariantBarcodes;
     }
+
+    public function setExportReferrerId($id): void
```

Comment:
We should have at least some idea what this value could or should be. Maybe it even makes sense to describe the value in a bit more detail here.

```suggestion
    /**
     * @var string|int|float $id
     */
    public function setExportReferrerId($id): void
```